executor/builder.go

```diff
@@ -95,6 +96,9 @@ type MockPhysicalPlan interface {
 }
 
 func (b *executorBuilder) build(p plannercore.Plan) Executor {
+	if config.GetGlobalConfig().EnableCollectExecutionInfo {
```

Comment:
add `b.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl == nil` check to avoid repeat assignment